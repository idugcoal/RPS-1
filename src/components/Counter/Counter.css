.counter {
  display: flex;
  flex-direction: column;
  align-items: center;
}

.name {
  font-size: 120%;
  margin-bottom: 5px;
}

.wins {
  font-size: 200%;
}
