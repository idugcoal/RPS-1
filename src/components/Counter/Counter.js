import React from 'react';
import PropTypes from 'prop-types';
import style from './Counter.css';

const ComputerCounter = ({ name, wins }) => (
  <div className={style.counter}>
    <div className={style.name}>{name}</div>
    <div className={style.wins}>{ wins }</div>
  </div>
);

ComputerCounter.propTypes = {
  name: PropTypes.string,
  wins: PropTypes.number,
};

export default ComputerCounter;
