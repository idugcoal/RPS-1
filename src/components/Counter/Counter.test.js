import React from 'react';
import { shallow } from 'enzyme';
import Counter from './Counter';

test('should render correctly with props', () => {
  const wrapper = shallow(<Counter
    name='You'
    wins={10}
  />);
  expect(wrapper).toMatchSnapshot();
});
