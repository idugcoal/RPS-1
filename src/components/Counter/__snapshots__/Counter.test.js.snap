// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly with props 1`] = `
<div>
  <div>
    You
  </div>
  <div>
    10
  </div>
</div>
`;
